config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
@AD7six `FILTER_VALIDATE_BOOLEAN` is working very nicely. It converts '0', 'false', 'FALSE', 'False', 'off' to `false`, it converts 'true', 'True', 'on', '1', to `true`. It is clearly better than other options.